docs/csharp/quick-starts/arrays-and-collections.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Quick Start - Collections - C# Guide
+description: Learn C# by exploring the List collection in this quick start.
+keywords: C#, Get Started, tutorial, collections, List
+author: billwagner
+ms.author: wiwagn
+ms.date: 10/13/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+---
+# C# Quick start: Collections #
```

Comment:
this is not hooked up to the TOC. Should it be or not?

Reply:
For now, I'm not adding it. I'd like to learn which has higher satisfaction in the user study: the local version, or the online / interactive version. Or, it may be both. 

I'll update after the user study, based on what we learn.